src/cljs/webchange/admin/schools/views.cljs

```diff
@@ -0,0 +1,65 @@
+(ns webchange.admin.schools.views
+  (:require [re-frame.core :as re-frame]
+            [webchange.editor-v2.components.card.views :refer [list-card] :as card]
+            [cljs-react-material-ui.reagent :as ui]
+            [cljs-react-material-ui.icons :as ic]))
+
+(re-frame/reg-sub
+  ::schools-list
+  (fn [db]
+    ;; (get-in db [:dashboard :schools])
+    ;;#############################
+    [{:id 1 :name "School 01" :presentation 37 :users 90}
+     {:id 2 :name "School 02" :presentation 37 :users 90}
+     {:id 3 :name "School 03" :presentation 37 :users 90}
+     {:id 4 :name "School 04" :presentation 37 :users 90}
+     ]
+    ;;############################
+    ))
+
+
+(defn school-item [data]
+  (let [list-styles (card/get-styles)]
+    (println data)
+    [ui/list-item
+     [ui/list-item-text {:primary (:name data)}]
+     [ui/list-item-secondary-action
+      [ui/button {:aria-label "Presentation"
+                  :style {:background-color "white"
```

Comment:
don't use inline styles. component styles should be defined in styles.scss file

Reply:
Yes, I'm aware some code should be in state.cljs and styles.scss, that's why I created the schools/styles.scss even though I'm not using it. This code was a prototype, Ivan asked me to create a draft with what I had at the end of the day even though it wasn't finished.